LondonBritishLibrary/orient/BLorient11601.xml

```diff
@@ -0,0 +1,305 @@
+                            </handDesc>
+                        </physDesc>
+                    </msPart>
+                    <msPart xml:id="p2"><msIdentifier/><msContents><summary/>
+                        <msItem xml:id="p2_i1">
+                            <locus from="77ra" to="82rb"/>
+                            <title ref="LIT3120Anapho"/>
+                            <incipit xml:lang="gez">አኰቴተ፡ ቍርባን፡ ዘዮሐንስ፡ ቅዱስ፡ አፈ፡ ወርቅ፡</incipit>
+                        </msItem> 
+                    </msContents>
+                        <physDesc>
+                            <objectDesc>
+                                <layoutDesc>  
+                                    <layout columns="2" writtenLines="20">
+                                        <locus from="77" to="82"></locus>
+                                        <note>Dimensions below refer to the size of a folio.</note>
+                                        <note>Number of written lines varies between 17 and 23.</note>
+                                        <dimensions unit="mm">
+                                            <height>135</height>
+                                            <width>120</width>
+                                        </dimensions>
+                                    </layout>
+                                </layoutDesc>
+                            </objectDesc>
+                            <handDesc>
+                                <handNote xml:id="h3" script="Ethiopic">
+                                    <locus from="77ra" to="82rb"/>
+                                    <date notBefore="1700" notAfter="1799"/>
+                                    <seg type="rubrication"/>
+                                </handNote>  
+                            </handDesc>
+                        </physDesc>
+                    </msPart>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Chronicles"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS14317Rowe" passive="BLorient11601"/>
```

Comment:
when you have a persName with `type="owner" `(or `"donor"` or `"scribe" `etc) there is actually no need for an extra relation

alternatively one can list all relevant persons inside of the `profileDesc `as

```
<particDesc>
               <listPerson>
                  <person>
                     <persName ref="PRS14317Rowe" role="owner"/>
                  </person>
                  <person>
                     <persName ref="XXXX" role="scribe"/>
                  </person>
                  <person>
                     <persName ref="XXXX" role="bequeather"/>
                     </person>

... and so on ...

                </listPerson>
                </particDesc>
```

see https://betamasaheft.eu/Guidelines/?id=particDescPersons

Reply:
If double information is a disadvantage, I prefer to delete the relation.